src/MessageContainer.js

```diff
@@ -99,66 +82,68 @@ export default class MessageContainer extends React.Component {
   }
 
   scrollTo(options) {
-    this._invertibleScrollViewRef.scrollTo(options);
+    this.refs.flatListRef.scrollToOffset(options)
   }
 
-  renderRow(message, sectionId, rowId) {
-    if (!message._id && message._id !== 0) {
-      console.warn('GiftedChat: `_id` is missing for message', JSON.stringify(message));
+  renderRow({item,index}) {
+    if (!item._id && item._id !== 0) {
+      console.warn('GiftedChat: `_id` is missing for message', JSON.stringify(item));
     }
-    if (!message.user) {
-      if (!message.system) {
-        console.warn("GiftedChat: `user` is missing for message", JSON.stringify(message));
+    if (!item.user) {
+      if (!item.system) {
+        console.warn("GiftedChat: `user` is missing for message", JSON.stringify(item));
       }
-      message.user = {};
+      item.user = {};
     }
 
     const messageProps = {
       ...this.props,
-      key: message._id,
-      currentMessage: message,
-      previousMessage: message.previousMessage,
-      nextMessage: message.nextMessage,
-      position: message.user._id === this.props.user._id ? 'right' : 'left',
+      key: item._id,
+      currentMessage: item,
+      previousMessage: item.previousMessage,
+      nextMessage: item.nextMessage,
+      position: item.user._id === this.props.user._id ? 'right' : 'left',
     };
 
     if (this.props.renderMessage) {
       return this.props.renderMessage(messageProps);
```

Comment:
Is it on purpose that this is outside of the `{ transform: [{ scaleY: -1 }] }` ? I don't think people will want to have to set that themselves, and when we go to use `invert={true}` on the container later, the people that do have their own `{ transform: [{ scaleY: -1 }] }` will have to remove it again and ughhhhh hahaha :)